core/src/main/java/org/apache/iceberg/rest/RESTObjectMapper.java

```diff
@@ -25,14 +25,14 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.databind.PropertyNamingStrategy;
 
-class RESTObjectMapper {
+public class RESTObjectMapper {
```

Comment:
I am able to keep this class as protected in src, but test (`TestlambdaRESTInvoker`) still needs to use it. Not sure if there is any way to test the client without changing this.

Reply:
If I'm not mistaken what's ultimately driving this is us injecting the `Mapper` into the RestClient. If we don't do that and instead keep that just static within the parent `RestClient` implementation I think we won't even need to expose this class. Is there a reason we want to pass in the `RestObjectMapper` to the client? (any reason to extend it/customize it?)